composer.json

```diff
@@ -32,7 +32,8 @@
         "symfony/framework-bundle": "^3.4 || ^4.0",
         "symfony/phpunit-bridge": "^3.4 || ^4.1",
         "symfony/twig-bundle": "^3.4 || ^4.0",
-        "twig/twig": "^1.35 || ^2.0"
+        "twig/twig": "^1.35 || ^2.0",
+        "symfony/web-link": "^3.4 || ^4.0"
```

Comment:
Shouldn't `fig/link-util` also be required since `GenericLinkProvider` and `Link` are directly used? Or is a transitive dependency acceptable in this case?